lib/dbservice_web/controllers/student_controller.ex

```diff
@@ -562,4 +573,89 @@ defmodule DbserviceWeb.StudentController do
       _ -> true
     end
   end
+
+  # function to verify student data with the values recieved in the verification params
+  swagger_path :verify_student do
+    post("/api/student/verify-student")
+
+    parameters do
+      body(:body, Schema.ref(:VerifyStudentRequest), "The verify student request", required: true)
```

Comment:
Update the message to read bette. Instead of "The verify student requrest", "parameters needed for student veritication" or something?

Reply:
Ok, I will update the message.